src/remotebuild/test/help.ts

```diff
@@ -0,0 +1,253 @@
+﻿/**
+ *******************************************************
+ *                                                     *
+ *   Copyright (C) Microsoft. All rights reserved.     *
+ *                                                     *
+ *******************************************************
+ */
+
+/// <reference path="../../typings/mocha.d.ts"/>
+/// <reference path="../../typings/should.d.ts"/>
+/// <reference path="../../typings/nconf.d.ts" />
+/// <reference path="../../typings/tacoUtils.d.ts"/>
+/// <reference path="../../typings/tacoTestsUtils.d.ts"/>
+
+"use strict";
+
+/* tslint:disable:no-var-requires */
+// var require needed for should module to work correctly
+// Note not import: We don't want to refer to shouldModule, but we need the require to occur since it modifies the prototype of Object.
+var shouldModule: any = require("should");
+/* tslint:enable:no-var-requires */
+
+/* tslint:disable:no-var-requires */
+// Special case to allow using color package with index signature for style rules
+var colors: any = require("colors/safe");
+/* tslint:enable:no-var-requires */
+
+import nconf = require("nconf");
+import path = require("path");
+import Q = require("q");
+
+import tacoUtils = require("taco-utils");
+import Help = require("../lib/help");
+import cli = require("../lib/cli");
+import RemoteBuildConf = require("../lib/remoteBuildConf");
+import tacoTestsUtils = require("taco-tests-utils");
+
+import UtilHelper = tacoUtils.UtilHelper;
+import ms = tacoTestsUtils;
+
+import ICommandData = tacoUtils.Commands.ICommandData;
+
+function parseRemoteBuildConf(): RemoteBuildConf {
+    // Configuration preference: command line, then anything in a config file if specified by the --config arg, then some defaults for options not yet set
+    nconf.argv();
```

Comment:
Is this appropriate for the tests? If we aren't going to mess around with process.argv then perhaps it would be better to go straight to using a config file, or nconf.overrides or something like that to specify particular arguments?